src/lib/international-telephone-codes.ts

```diff
@@ -0,0 +1,1275 @@
+  },
+  LA: {
+    code: '+856',
+    unicodeFlag: '🇱🇦',
+    svgFlag: require('../../assets/icons/flags/LA.svg'),
+  },
+  TW: {
+    code: '+886',
+    unicodeFlag: '🇹🇼',
+    svgFlag: require('../../assets/icons/flags/TW.svg'),
+  },
+  CG: {
+    code: '+242',
+    unicodeFlag: '🇨🇬',
+    svgFlag: require('../../assets/icons/flags/CG.svg'),
+  },
+  CZ: {
+    code: '+420',
+    unicodeFlag: '🇨🇿',
+    svgFlag: require('../../assets/icons/flags/CZ.svg'),
+  },
+  GB: {
+    code: '+44',
+    unicodeFlag: '🇬🇧',
+    svgFlag: require('../../assets/icons/flags/GB.svg'),
+  },
+  NE: {
+    code: '+227',
+    unicodeFlag: '🇳🇪',
+    svgFlag: require('../../assets/icons/flags/NE.svg'),
+  },
+  CD: {
+    code: '+243',
+    unicodeFlag: '🇨🇩',
+    svgFlag: require('../../assets/icons/flags/CD.svg'),
+  },
+  BS: {
+    code: '+1 242',
+    unicodeFlag: '🇧🇸',
+    svgFlag: require('../../assets/icons/flags/BS.svg'),
+  },
+  CC: {
+    code: '+61 891',
+    unicodeFlag: '🇨🇨',
+    svgFlag: require('../../assets/icons/flags/CC.svg'),
+  },
+  CF: {
+    code: '+236',
+    unicodeFlag: '🇨🇫',
+    svgFlag: require('../../assets/icons/flags/CF.svg'),
+  },
+  CK: {
+    code: '+682',
+    unicodeFlag: '🇨🇰',
+    svgFlag: require('../../assets/icons/flags/CK.svg'),
+  },
+  FK: {
+    code: '+500',
+    unicodeFlag: '🇫🇰',
+    svgFlag: require('../../assets/icons/flags/FK.svg'),
+  },
+  FO: {
+    code: '+298',
+    unicodeFlag: '🇫🇴',
+    svgFlag: require('../../assets/icons/flags/FO.svg'),
+  },
+  HM: {
+    code: '+672',
+    unicodeFlag: '🇭🇲',
+    svgFlag: require('../../assets/icons/flags/HM.svg'),
+  },
+  KM: {
+    code: '+269',
+    unicodeFlag: '🇰🇲',
+    svgFlag: require('../../assets/icons/flags/KM.svg'),
+  },
+  KY: {
+    code: '+1 345',
+    unicodeFlag: '🇰🇾',
+    svgFlag: require('../../assets/icons/flags/KY.svg'),
+  },
+  MH: {
+    code: '+692',
+    unicodeFlag: '🇲🇭',
+    svgFlag: require('../../assets/icons/flags/MH.svg'),
+  },
+  MP: {
+    code: '+1 670',
+    unicodeFlag: '🇲🇵',
+    svgFlag: require('../../assets/icons/flags/MP.svg'),
+  },
+  TC: {
+    code: '+1 649',
+    unicodeFlag: '🇹🇨',
+    svgFlag: require('../../assets/icons/flags/TC.svg'),
+  },
+  TF: {
+    code: '+672',
+    unicodeFlag: '🇹🇫',
+    svgFlag: require('../../assets/icons/flags/TF.svg'),
+  },
+  UM: {
+    code: '+1',
+    unicodeFlag: '🇺🇲',
+    svgFlag: require('../../assets/icons/flags/UM.svg'),
+  },
+  VA: {
+    code: '+39',
+    unicodeFlag: '🇻🇦',
+    svgFlag: require('../../assets/icons/flags/VA.svg'),
+  },
+  XK: {
+    code: '+383',
+    unicodeFlag: '🇽🇰',
+    svgFlag: require('../../assets/icons/flags/XK.svg'),
+  },
+}
+
+const DEFAULT_PHONE_COUNTRY = 'US'
+
+export function getDefaultCountry(location?: {countryCode?: string}) {
+  const supportedCountries = Object.keys(INTERNATIONAL_TELEPHONE_CODES)
+
+  if (
+    location?.countryCode &&
+    supportedCountries.includes(location.countryCode.toUpperCase())
```

Comment:
Could just do `location.countryCode.toUpperCase() in INTERNATIONAL_TELEPHONE_CODES` or `INTERNATIONAL_TELEPHONE_CODES[cc]` and skip the array scan. You do this below already :P